--- lib/libmedida/src/medida/histogram.cc	2021-12-16 16:57:02.652813232 +0100
+++ lib/libmedida/src/medida/histogram.cc	2023-02-18 02:37:19.393719526 +0100
@@ -7,6 +7,7 @@
 #include <cmath>
 #include <mutex>
 #include <algorithm>
+#include <stdexcept>

 #include "medida/stats/exp_decay_sample.h"
 #include "medida/stats/uniform_sample.h"

--- src/historywork/Progress.cpp
+++ src/historywork/Progress.cpp
@@ -2,6 +2,7 @@
 // under the Apache License, Version 2.0. See the COPYING file at the root
 // of this distribution or at http://www.apache.org/licenses/LICENSE-2.0

+#include <stdint.h>
 #include "historywork/Progress.h"
 #include "history/HistoryManager.h"
 #include "ledger/LedgerRange.h"

--- src/invariant/Invariant.h
+++ src/invariant/Invariant.h
@@ -4,6 +4,7 @@
 // under the Apache License, Version 2.0. See the COPYING file at the root
 // of this distribution or at http://www.apache.org/licenses/LICENSE-2.0

+#include <xdrpp/types.h>
 #include <functional>
 #include <memory>
 #include <string>
